scripts/am

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Always use on a separate merge-only repo!
+
+# Best used with a wrapper script in your PATH
+# #!/bin/bash
+# export USERNAME=<github username>
+# export PERSONAL_TOKEN=<github personal token from https://github.com/settings/tokens>
+# <absolute path to brooklyn-dist>/scripts/am $@
+
+set -e
+
+PR=$1
+: ${PR_REMOTE:=upstream}
+: ${APACHE_REMOTE:=apache}
+: ${CLOSES_MESSAGE:="Closes #"}
+
+REMOTE_URL=$(git remote get-url ${APACHE_REMOTE})
+MODULE=${REMOTE_URL##*/}
+
+if [ -z "$PR" ]; then
+    echo "Usage: am <PR number>"
+    exit 1
+fi
+
+if ! git branch | grep -q "* master"; then
+    echo "Must be on master branch"
+    exit 1
+fi
```

Comment:
This just makes sure we are on `master` (vs say `0.10.0`). Updating to remote master happens here https://github.com/apache/brooklyn-dist/pull/74/files#diff-7c4ab39c149b092f17e43d7f46616063R85.